docs/source/using/index.adoc

```diff
@@ -1,14 +1,12 @@
-[[using-index]]
+[[using-minishift]]
 = Using Minishift
```

Comment:
Let's defer this for a bit, if possible. I think we go with managing we would need to say something like "Managing the Minishift VM". For now I suggest to keep it, but let's keep this in mind.